src/js/providers/video-listener-mixin.js

```diff
@@ -177,7 +177,7 @@ const VideoListenerMixin = {
 
         this.trigger(
             MEDIA_ERROR,
-            new PlayerError(`Error loading media ${errorData[0]}`, code, this.video.error)
+            new PlayerError(MSG_TECHNICAL_ERROR, code, this.video.error)
```

Comment:
I think each error code should have a specific message; 
'File could not be played' -> `cantPlayVideo `
'Unknown decode error' -> `cantPlayVideo `
the rest sounds like `MSG_TECHNICAL_ERROR `